_overviews/scala3-book/taste-indentation.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Indentation
+type: section
+description: This section demonstrates Scala 3 Indentation.
+num: 8
+previous-page: taste-vars-data-types
+next-page: taste-control-structures
+---
+
+Indentation refers to the increase or decrease of space between the left and right margin of a paragraph.
+In some programming languages, tab key is used to make indentations. This is not the case in Scala. Tab key automatically creates four spaces which Scala doesn't support.
+In Scala each level of indentation is 2 spaces.  Thus, instead of
```

Comment:
Actually, this is not a requirement, but you could say that this is common practice.